plugin-management-cli/src/test/resources/plugins.txt

```diff
@@ -0,0 +1,8 @@
+git:   latest
+
+job-import-plugin:2.1
+     #   should not be read
+#structs:1.6
+docker
```

Comment:
would be good to test url support as well, either in this file or another test

Reply:
Yeah, I agree. I added a question to the gitter channel about using ":" as a separator since urls will also have ":" in http://.... I was going to fix/change that before adding tests for it.